.buildkite/pipeline.yml

```diff
@@ -81,11 +81,11 @@ steps:
     if: |
       // Only run if:
       // 1. There's a tag and it's of the form 'v*' or 'operator/v*'
-      // 2. Source branch is main or release/*
+      // 2. Source branch is main or release/* (during release the branch will be equal to tag name)
       // 3. NIGHTLY_RELEASE envvar isn't set (paranoid check)
       // 4. This isn't a pull request (paranoid check)
       (build.tag =~ /^v/ || build.tag =~ /^operator\/v/) &&
-      (build.branch == "main" || build.branch =~ /^release\//) &&
+      (build.branch == "main" || build.branch =~ /^release\// || build.branch =~ /^v/ || build.branch =~ /^operator\/v/) &&
```

Comment:
Can we remove the checks on `main` and `release/` as we know they'll never be relevant? We could alternatively remove all branch level checks as they're synonymous with `tag` and specifying both may be misleading.

Reply:
Agree removed branch check